dbt/adapters/spark_cde/cdeapisession.py

```diff
@@ -699,6 +699,8 @@ def get_auth_endpoint(self):
         return self.get_base_auth_url() + "gateway/authtkn/knoxtoken/api/v1/token"
 
     def connect(self, user_name, password, base_auth_url, base_api_url, session_params, verify_ssl_certificate=False):
+        global DEFAULT_CDE_JOB_TIMEOUT, DEFAULT_POLL_WAIT, DEFAULT_LOG_WAIT
```

Comment:
Looks good. Can we add some default values here in case user forgets to add.

Reply:
defaults are already added in L41-48.